Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -51,9 +50,32 @@
 namespace WebKit {
 using namespace WebCore;
 
-UIScrollView *RemoteScrollingCoordinatorProxy::scrollViewForScrollingNodeID(ScrollingNodeID nodeID) const
+RemoteScrollingCoordinatorProxyIOS::RemoteScrollingCoordinatorProxyIOS(WebPageProxy& webPageProxy)
+    : RemoteScrollingCoordinatorProxy(webPageProxy)
 {
-    auto* treeNode = m_scrollingTree->nodeForID(nodeID);
+}
+
+OptionSet<TouchAction> RemoteScrollingCoordinatorProxyIOS::activeTouchActionsForTouchIdentifier(unsigned touchIdentifier) const
+{
+    auto iterator = m_touchActionsByTouchIdentifier.find(touchIdentifier);
+    if (iterator == m_touchActionsByTouchIdentifier.end())
+        return { };
+    return iterator->value;
```

Comment:
Would `m_touchActionsByTouchIdentifier.get(touchIdentifier);` work here? I think the return value of `.get()` if the identifier is not found should be an empty OptionSet.